badgers-web/src/app/page.tsx

```diff
@@ -180,6 +180,14 @@ export default function Home() {
                     <Row name="Types" path="/npm/types/:pkg" inject={['react']} />
                     <Row name="Types (scoped)" path="/npm/types/:org/:pkg" inject={['@octokit', 'rest']} />
                 </div>
+              </Section>
+              <Section name="PyPI">
+                <div className="grid grid-cols-[auto_1fr_auto] items-center gap-x-8">
+                    <Row name="Name" path="/pypi/name/:pkg" inject={['numpy']} />
+                    <Row name="Version" path="/pypi/version/:pkg" inject={['numpy']} />
+                    <Row name="Name and Version" path="/pypi/info/:pkg" inject={['numpy']} />
+                    <Row name="License" path="/pypi/license/:pkg" inject={['numpy']} />
```

Comment:
We could use a "lighter" package here to speed up loading of the home page.

```suggestion
                    <Row name="Name" path="/pypi/name/:pkg" inject={['docopt-ng']} />
                    <Row name="Version" path="/pypi/version/:pkg" inject={['docopt-ng']} />
                    <Row name="Name and Version" path="/pypi/info/:pkg" inject={['docopt-ng']} />
                    <Row name="License" path="/pypi/license/:pkg" inject={['docopt-ng']} />
```